src/game/game/game.c

```diff
@@ -202,21 +206,42 @@ GAMEFLOW_OPTION Game_Stop(void)
     if (g_LevelComplete) {
         // TODO: why is this made unavailable?
         g_GameInfo.current[g_CurrentLevel].flags.available = 0;
-        return GF_LEVEL_COMPLETE | g_GameInfo.select_level_num;
+        return (GAMEFLOW_INSTRUCTION) {
+            .instruction = GF_LEVEL_COMPLETE,
+            .param = g_GameInfo.select_level_num,
```

Comment:
I'm confused what is going on here with `select_level_num`. This function is kinda confusing overall, but I left it alone since it's working and it's unrelated to this PR.